python/tk_multi_reviewsubmission/renderer.py

```diff
@@ -65,8 +70,14 @@ def gather_nuke_render_info(self, path, output_path,
         writenode_quicktime_settings = self.__app.execute_hook_method("codec_settings_hook",
                                                                       "get_quicktime_settings")
 
-        render_script_path = os.path.join(self.__app.disk_location, "hooks",
-                                          "nuke_batch_render_movie.py")
+        # making the python script passed to nuke configurable as a setting because
+        # making it a hook would still not allow us to subprocess it out
+        render_script_template = self.__app.get_template("render_script")
```

Comment:
Need to handle the case when this is undefined for backwards compatibility

Reply:
Added allows_empty for this and burnin_nk (introduced newly from vanilla shotgun) and handling it now.